src/main/java/io/khasang/jrepetitor/controller/AppController.java

```diff
@@ -14,12 +14,9 @@
     @Autowired
     private CreateTable createTable;
 
-    // Фабричный метод бинов.
-    // Cat cat = new Cat();
-
     @RequestMapping("/")
     public String helloPage(){
-        return "cat";
+        return "hello";
```

Comment:
you can't change other logic outside your ticket

Reply:
done (local)